man/man8/zpool.8

```diff
@@ -272,6 +272,16 @@ However, raidz vdev types are not supported for the intent log.
 For more information, see the
 .Sx Intent Log
 section.
+.It Sy special
+A special device dedicates the device to only participate in the special
+allocation class. The redundancy of this device needs to match the
```

Comment:
I don't think "solely for allocation classes" really explains much here about what this is, since it is mostly a circular definition here.  Maybe something more clear like:

    A device dedicated solely for allocating various kinds of internal metadata,
    and optionally small file data.

Reply:
Since metadata will also be double/triple ditto'd in addition to VDEV mirroring, it isn't clear whether there is a need for e.g. triple mirrors to match the allowed two failures of e.g. RAID-Z2 of regular VDEVs?

I'd think 'needs to match' is a bit strong, probably 'should match ... in the pool, in order to maintain adequate redundancy.' is better?